docs/Install/IP_address_of_host_systems.md

```diff
@@ -1,4 +1,9 @@
 # <a name="GUID-1D2971AF-D537-4521-ACAA-D910F50C43E0"/> IP address of host systems
 
-By default, Swarm Learning uses the host network to connect and communicate with its peers. In this case, the IP addresses represent the IP addresses or a fully qualified domain name \(FQDN\) of the host systems on which the containers are run. Swarm Learning can be configured to use a Docker bridge network. In this case, the IP addresses represent the IP addresses or FQDN of the containers themselves.
+The `--host-ip` and `slhostip` IP addresses in the run scripts and the SWOP profile are the IP addresses of the host machine, where the respective containers are running on the host machine. Based on access, user can even use the FQDN of the host system.
```

Comment:
The `--host-ip` and `slhostip` IP addresses in the run scripts and the SWOP profile are the IP addresses of the host machine, where the respective containers are running. Based on access, user can even use the FQDN of the host system

Reply:
Resolved